variables.tf

```diff
@@ -1,3 +1,13 @@
+variable "ip_addresses" {
+  type        = set(string)
+  description = "The list of IP addresses that belong to the IP Group."
+
+  validation {
```

Comment:
Should we check for valid ip addresses too? Does it support IPv4 only or IPv6 too?